### PR TITLE
change method name convention

### PR DESCRIPTION
**Pull request by weilix88**

### CODE REVIEW

File: BuildSimHubAPI/helpers/parametric_model.py [hunk 1]
```diff
@@ -12,9 +12,9 @@ class ParametricModel:
     BASE_URL = 'https://my.buildsim.io/'
 
     def __init__(self, userKey, model_key, base_url=None):
```
Comment (mostaphaRoudsari) [inline BuildSimHubAPI/helpers/parametric_model.py]: Hi @weilix88, This looks great. You're missing `userKey` input here which should be `user_key`.

File: BuildSimHubAPI/helpers/parametric_job.py [hunk 2]
```diff
@@ -9,22 +9,22 @@ class ParametricJob():
     BASE_URL = 'https://my.buildsim.io/'
 
     def __init__(self, userKey, mk, base_url=None):
```
Comment (mostaphaRoudsari) [inline BuildSimHubAPI/helpers/parametric_job.py]: `mk` is not consistent between the classes. In `/helpers/parametric_model.py` it is called `model_key`.

Comment (weilix88) [reply BuildSimHubAPI/helpers/parametric_job.py]: oh, yes, I will change it and double check other parameters, thanks for pointing it out!

File: BuildSimHubAPI/helpers/parametric_model.py (hunk 1 above)
Comment (weilix88) [reply BuildSimHubAPI/helpers/parametric_model.py]: will fix that, thanks!

**Pull request merged**